util/missingcrypto.txt

```diff
@@ -627,14 +626,9 @@ ERR_load_DSO_strings(3)
 ERR_load_EC_strings(3)
 ERR_load_ENGINE_strings(3)
 ERR_load_ERR_strings(3)
-ERR_load_ESS_strings(3)
 ERR_load_EVP_strings(3)
-ERR_load_HTTP_strings(3)
-ERR_load_KDF_strings(3)
```

Comment:
Errr - no? This remains available and undocumented?

Reply:
Considering this comment, I suppose this is no longer a mystery: https://github.com/openssl/openssl/pull/13463#issuecomment-731171065

Reply:
That comment explains the change in missingcrypto111.txt, but not this one. `ERR_load_KDF_strings()` remains as a function that is available in 3.0; it is undocumented; it has been undocumented for a long time (since 1.1.1). Therefore it should not be removed from this file. Only when we've documented it (or its been removed as a public function) should it be removed from here.

Reply:
Huh, ok, so it should exist in both files then, that's what you're saying?

Reply:
Added back

Reply:
Yes! Thanks.